src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7574,6 +7574,63 @@ public enum NormalizationForm
         FormKC = 5,
         FormKD = 6,
     }
+    public readonly struct Rune : IComparable<Rune>, IEquatable<Rune>
+    {
+        private readonly int _dummyPrimitive;
+        public Rune(char ch) { throw null; }
+        public Rune(int value) { throw null; }
+        [CLSCompliant(false)]
+        public Rune(uint value) { throw null; }
+        public static bool operator ==(Rune left, Rune right) { throw null; }
+        public static bool operator !=(Rune left, Rune right) { throw null; }
+        public static bool operator <(Rune left, Rune right) { throw null; }
+        public static bool operator <=(Rune left, Rune right) { throw null; }
+        public static bool operator >(Rune left, Rune right) { throw null; }
+        public static bool operator >=(Rune left, Rune right) { throw null; }
+        public static explicit operator Rune(char ch) { throw null; }
+        [CLSCompliant(false)]
+        public static explicit operator Rune(uint value) { throw null; }
+        public static explicit operator Rune(int value) { throw null; }
+        public bool IsAscii { get { throw null; } }
+        public bool IsBmp { get { throw null; } }
+        public int Plane { get { throw null; } }
```

Comment:
Should this be more descriptive: `UnicodePlane`?

Reply:
I think current version should be fine considering the whole class is related to the Unicode code point